src/test/miner_tests.cpp

```diff
@@ -23,6 +23,7 @@
 #include <util/system.h>
 #include <util/time.h>
 #include <validation.h>
+#include <versionbits.h>
```

Comment:
22630: this fixes an issue introduced in 22277 which we don't have yet, should probably postpone it till later